backend/tests/integration/test_b07_p8_topology_closure_pack.py

```
@@ -0,0 +1,498 @@
+    return resolved
+
+
+def _validate_payload(schema: dict[str, Any], payload: Any) -> None:
+    Draft202012Validator(schema).validate(payload)
+
+
+def _seed_tenant(runtime_db_url: str) -> _TenantFixture:
+    tenant_id = uuid4()
+    tenant_key = f"b07_p8_tenant_key_{tenant_id.hex[:10]}"
+    api_key_hash = hashlib.sha256(tenant_key.encode("utf-8")).hexdigest()
+    stripe_secret = "whsec_b07_p8_stripe"
```

Ruff (0.15.0)

[error] Possible hardcoded password assigned to: "stripe_secret"

(S105)